queries/java/highlights.scm

```diff
@@ -59,61 +60,79 @@
 (hex_floating_point_literal) @number
 (character_literal) @string
 (string_literal) @string
-(true) @constant.builtin
-(false) @constant.builtin
 (null_literal) @constant.builtin
 
 (comment) @comment
 
+[
+"true"
+"false"
+] @boolean
+
 ; Keywords
 
-"abstract" @keyword
-"assert" @keyword
-"break" @keyword
-"case" @keyword
-"catch" @keyword
-"class" @keyword
-"continue" @keyword
-"default" @keyword
-"do" @keyword
-"else" @keyword
-"enum" @keyword
-"exports" @keyword
-"extends" @keyword
-"final" @keyword
-"finally" @keyword
-"for" @keyword
-"if" @keyword
-"implements" @keyword
-"import" @keyword
-"instanceof" @keyword
-"interface" @keyword
-"module" @keyword
-"native" @keyword
-"new" @keyword
-"open" @keyword
-"opens" @keyword
-"package" @keyword
-"private" @keyword
-"protected" @keyword
-"provides" @keyword
-"public" @keyword
-"requires" @keyword
-"return" @keyword
-"static" @keyword
-"strictfp" @keyword
-"switch" @keyword
-"synchronized" @keyword
-"throw" @keyword
-"throws" @keyword
-"to" @keyword
-"transient" @keyword
-"transitive" @keyword
-"try" @keyword
-"uses" @keyword
-"volatile" @keyword
-"while" @keyword
-"with" @keyword
+[
+"abstract"
+"assert"
+"break"
+"case"
+"catch"
+"class"
+"continue"
+"default"
+"do"
+"enum"
+"exports"
+"extends"
+"final"
+"finally"
+"implements"
+"instanceof"
+"interface"
+"module"
+"native"
+"new"
+"open"
+"opens"
+"package"
+"private"
+"protected"
+"provides"
+"public"
+"requires"
+"return"
+"static"
+"strictfp"
+"switch"
```

Comment:
Hmm... You probably want to mark this as conditionnal instead ?

Reply:
Lemme see java docs

Reply:
Oh you meant only `switch`
Wait

Reply:
Done

Reply:
And maybe `case` and such

Reply:
Oh shoot, sorry! wait

Reply:
Done